presto-main/src/main/java/com/facebook/presto/operator/TaskContext.java

```diff
@@ -90,7 +89,20 @@
     @GuardedBy("cumulativeMemoryLock")
     private long lastTaskStatCallNanos = 0;
 
-    public TaskContext(QueryContext queryContext,
+    public static TaskContext getInstance(QueryContext queryContext,
```

Comment:
Call it `createTaskContext` instead.

Put first argument on its own line. Putting it on the same line is legacy code style. Move it to a new line whenever you modify those code.